docs/sriov.md

```diff
@@ -144,7 +145,10 @@ Once the cluster is deployed, we can move to SR-IOV specific components.
 
 # Deploy SR-IOV services
 
-First, deploy Multus with default Flannel backend:
+First, deploy latest Multus with default Flannel backend. We will need to build
```

Comment:
there is an image uploaded to dockerhub that includes the necessary changes. Need to change wording here to reflect that.